### ISSUE TITLE
[a11y] "Find in page" edit area is not announced

### ISSUE THREAD

**Issue by lobontiumira** [mozilla-mobile/focus-android#6133]
## Steps to reproduce
1. Turn talkback/screen reader on.
2. Load a page.
3. Tap on the three-dot menu, and select the "Find in page" option using talkback gestures.
4. Listen what the screen reader announces.

### Expected behavior
The screen reader announces the "Find in page" edit area.

### Actual behavior
The screen reader announces the keyboard, then the menu that is under the "Find in page" edit area, instead the "close" button.
If the user taps on the selected area (please see the screenshot below), the screen reader announces "Dismiss Find in page button. Double tap to activate."

### Device information
* Android device: Google Pixel (Android 10)
* Focus version: Beta 96.0.0-beta.5
* In Fenix, the screen reader focuses the edit area and announces it.

![a11y](https://user-images.githubusercontent.com/40694158/147551710-c13ec21a-e42c-469e-b486-0f568fb6aa43.png)

**Comment by lobontiumira** [comment]
Reproducible on Focus Nightly 104.0a1 from 7/18 with Oppo Find X3 Lite (Android 11).

**Comment by lobontiumira** [comment]
Reproducible on the latest Focus Nightly 108.0a1 from 11/2 with Oppo Find X3 Lite (Android 11).

**Comment by cpeterso** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1802584

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
